plugins/modules/copr.py

```diff
@@ -52,6 +52,10 @@
               for example V(epel-7-x86_64). Default chroot is determined by the operating system,
               version of the operating system, and architecture on which the module is run.
         type: str
+    includepkgs:
+        description: List of packages to include.
+        required: false
+        type: str
```

Comment:
Please add this attribute.
```suggestion
        type: str
        version_added: 9.4.0
```

Reply:
Done